docs/runtime.md

```diff
@@ -72,6 +80,14 @@ $ cd runtime/target
 $ java -jar xpanse-runtime-1.0.0-SNAPSHOT.jar --spring.profiles.active=zitadel,zitadel-testbed
 ```
 
+By default, the Xpanse Runtime starts using local terraform. If you want to use the remote
```

Comment:
@WangLiNaruto we have this implemented in opposite way. Default is the terraform-boot. For developers who want to use the local, then they must remove 'terraform-boot' from the active profiles list.